### PR TITLE
Add configuration for page mode and displaying the first page as a single page

### LINKED ISSUE #225: [Android] pageMode is ignored
# Details
When passing configuration details to `PSPDFKitView`, specifically `pageMode="single"` this configuration is ignored. Is it not implemented on Android or is there some issue?

# Acceptance Criteria
Can configure `PSPDFKitView` to display a pdf as a single page.

### PR DESCRIPTION
**Pull request by irgendeinich**
## Resolves #225 

# Details

We were missing `pageMode` and related `firstPageAlwaysSingle` in the `ConfigurationAdapter` this adds those missing properties.

# Acceptance Criteria

- [x] When approved, right before merging, rebase with master and increment the package version in `package.json`, `package-lock.json`, and `samples/Catalog/package.json` (see example commit:  https://github.com/PSPDFKit/react-native/pull/202/commits/1bf805feef2ac268743e4905d94d6d8c8f16ec59).
- [ ] Create a new release (and tag) with the new package version (see https://github.com/PSPDFKit/react-native/releases).

### CODE REVIEW

File: android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java [hunk 1]
```diff
@@ -134,6 +139,12 @@ public ConfigurationAdapter(@NonNull Context context, ReadableMap configuration)
             if (configuration.hasKey(SHOW_DOCUMENT_TITLE_OVERLAY)) {
                 configureShowDocumentTitleOverlay(configuration.getBoolean(SHOW_DOCUMENT_TITLE_OVERLAY));
             }
+            if (configuration.hasKey(PAGE_MODE)) {
+                configurePageMode(configuration.getString(PAGE_MODE));
+            }
+            if (configuration.hasKey(FIRST_PAGE_ALWAYS_SINGLE)) {
+                configureFirstPageAlwaysSingle(configuration.getBoolean(FIRST_PAGE_ALWAYS_SINGLE));
```
Comment (davidschreiber) [inline android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java]: What is the default  of `configuration.getBoolean(...)`. What does it return if the value that was set is not a `boolean`?

File: android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java [hunk 2]
```diff
@@ -294,6 +305,22 @@ private void configureShowDocumentTitleOverlay(boolean showDocumentTitleOverlay)
         }
     }
 
+    private void configurePageMode(String pageMode) {
```
[the remark this answers is not in the document]
Comment (davidschreiber) [reply android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java]: Also, these parameters can be marked `final`.

[the remark this answers is not in the document]
Comment (irgendeinich) [reply android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java]: It is nullable, I'll add an extra case to use the default value when it's `null`.

File: android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java (hunk 1 above)
Comment (irgendeinich) [reply android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java]: If it's not a `boolean` an exception is thrown, this makes sense in my opinion so users know if they passed an invalid value in the JS code.

**Pull request merged**